### DISCUSSION TITLE
[FEATURE] Upload to Modrinth too

### DISCUSSION THREAD

**Discussion by Small-Ku** [mrjesen/ImBlockerFabric#46] [Ideas]
**Type**
Upload platform

**Description**
Add another mod hosting platform for mod downloading

**Usage**
People may download the mod with another platform.

**Additional context**
[Packwiz](https://github.com/packwiz/packwiz/) can use Modrinth and CurseForge as mod sources.
Also, at this moment, CurseForge is down with a runtime error.

**Comment by mrjesen** [comment]
Thank you, I will consider it

**Comment by mrjesen** [comment]
https://modrinth.com/mod/imblocker
done
